### DISCUSSION TITLE
Redash - String Split

### DISCUSSION THREAD

**Discussion by nagendran81101** [getredash/redash#6414] [Q&A]
How Can I split strings which is one column to multiple rows in redash8.0 Postgresql?.  I have tried split_part,string_to_array and string_to_table fuctions. It is not supporting in my version. Please help me find a way

**Comment by justinclift** [comment]
As a general concept, it *might* be better to ask that on the PostgreSQL support channels as it's likely to be a PostgreSQL string or array function.

PostgreSQL have Community based Slack, Discord, mailing lists, and other options. :smile:

* Slack: https://join.slack.com/t/postgresteam/shared_invite/zt-1qj14i9sj-E9WqIFlvcOiHsEk2yFEMjA
* Discord: https://discord.com/invite/bW2hsax8We
* Mailing lists: https://www.postgresql.org/list/
